### PR TITLE
Docs(web-twig): How to publish changes to package readonly repository

### PR DESCRIPTION
**Pull request by literat**

### CODE REVIEW

File: packages/web-twig/CONTRIBUTING.md [hunk 1]
```diff
@@ -65,4 +65,24 @@ Run `composer phpunit:update` or `make phpunit-update` to create or update snaps
...
-- run script `. ./newVersion.sh <version>`
+TODO: [Automate release process](https://github.com/lmc-eu/spirit-design-system/issues/393).
+
+As we are now using only `dev-main` version of this package, until we have stable release process, we are not able to release new version with release script and tag it.
+
+- Do not run script `. ./newVersion.sh <version>`
+
+### Monorepo and Packagist.org
+
+Packagist require a single repository for a single package to be present to publish the package. In order to publish this package we are using READ-ONLY repository https://github.com/lmc-eu/spirit-web-twig-bundle where we are publishing the subtree of this monorepo.
```
Comment (adamkudrna) [inline packages/web-twig/CONTRIBUTING.md:76]:
```suggestion
Packagist requires a single repository for a single package to be present to publish the package. In order to publish this package we are using READ-ONLY repository https://github.com/lmc-eu/spirit-web-twig-bundle where we are publishing the subtree of this monorepo.
```

File: packages/web-twig/CONTRIBUTING.md [hunk 2]
```diff
@@ -65,4 +65,24 @@ Run `composer phpunit:update` or `make phpunit-update` to create or update snaps
 
 ## Release new version
 
-- run script `. ./newVersion.sh <version>`
+TODO: [Automate release process](https://github.com/lmc-eu/spirit-design-system/issues/393).
+
+As we are now using only `dev-main` version of this package, until we have stable release process, we are not able to release new version with release script and tag it.
+
+- Do not run script `. ./newVersion.sh <version>`
```
Comment (adamkudrna) [inline packages/web-twig/CONTRIBUTING.md:72]: Could you either join it with the previous sentence, or maybe emphasize it if it deserves attention? It's weird to see it as a single-item list… 🤔 

```suggestion
As we are now using only `dev-main` version of this package, until we have stable release process, we are not able to release new version with release script and tag it. **Do not run the script `./newVersion.sh <version>`.**
```

or

```suggestion
As we are now using only `dev-main` version of this package, until we have stable release process, we are not able to release new version with release script and tag it.

⚠️ **DO NOT RUN** the script `./newVersion.sh <version>`!
```

**Review by adamkudrna — APPROVED** [review]

File: packages/web-twig/CONTRIBUTING.md (hunk 1 above)
Comment (literat) [reply packages/web-twig/CONTRIBUTING.md]: Fixed.

File: packages/web-twig/CONTRIBUTING.md (hunk 2 above)
Comment (literat) [reply packages/web-twig/CONTRIBUTING.md]: Second choice. Thanks.

**Pull request merged**